wagtailio/templates/includes/svg_sprites.html

```diff
@@ -15,4 +16,14 @@
.5923ZM36 0C25.2696 0 16.1755 6.50378 12.1181 15.5923L13.9444 16.4077C17.6995 7.99622 26.1054 2 36 2V0ZM13.0312 15H11V17H13.0312V15ZM11 15C9.00178 15 7.5 16.7383 7.5 18.5H9.5C9.5 17.7617 10.1857 17 11 17V15ZM7.5 18.5V33.5H9.5V18.5H7.5ZM7.5 33.5C7.5 35.4585 9.04147 37 11 37V35C10.146 35 9.5 34.354 9.5 33.5H7.5ZM18.25 22.25C18.25 19.5205 20.9138 17 24.75 17V15C20.1487 15 16.25 18.1045 16.25 22.25H18.25ZM24.75 17H47.25V15H24.75V17ZM47.25 17C50.9445 17 53.75 19.5326 53.75 22.25H55.75C55.75 18.0924 51.6805 15 47.25 15V17ZM53.75 22.25V26H55.75V22.25H53.75ZM53.75 26C53.75 33.729 47.479 40 39.75 40V42C48.5835 42 55.75 34.8335 55.75 26H53.75ZM39.75 40H32.25V42H39.75V40ZM32.25 40C24.521 40 18.25 33.729 18.25 26H16.25C16.25 34.8335 23.4165 42 32.25 42V40ZM18.25 26V22.25H16.25V26H18.25ZM29.4487 33.8162L31.3237 28.1912L29.4263 27.5588L27.5513 33.1838L29.4487 33.8162ZM30.6912 28.8237L36.3162 26.9487L35.6838 25.0513L30.0588 26.9263L30.6912 28.8237ZM36.3162 25.0513L30.6912 23.1763L30.0588 25.0737L35.6838 26.9487L36.3162 25.0513ZM31.3237 23.8088L29.4487 18.1838L27.5513 18.8162L29.4263 24.4412L31.3237 23.8088ZM27.5513 18.1838L25.6763 23.8088L27.5737 24.4412L29.4487 18.8162L27.5513 18.1838ZM26.3088 23.1763L20.6838 25.0513L21.3162 26.9487L26.9412 25.0737L26.3088 23.1763ZM20.6838 26.9487L26.3088 28.8237L26.9412 26.9263L21.3162 25.0513L20.6838 26.9487ZM25.6763 28.1912L27.5513 33.8162L29.4487 33.1838L27.5737 27.5588L25.6763 28.1912ZM51.5587 50.4677C47.0235 53.34 41.7391 55 36 55V57C42.1359 57 47.789 55.2225 52.6288 52.1573L51.5587 50.4677ZM36 55C30.102 55 24.8179 53.3385 20.2851 50.4677L19.2149 52.1573C24.0571 55.224 29.7105 57 36 57V55ZM19.6476 50.3178C8.52024 51.4632 0 60.636 0 72.0938H2C2 61.6765 9.72976 53.3493 19.8524 52.3072L19.6476 50.3178ZM0 72.0938V73.5H2V72.0938H0ZM0 73.5C0 78.271 3.72897 82 8.5 82V80C4.83353 80 2 77.1665 2 73.5H0ZM8.5 82H21V80H8.5V82ZM22 81V71H20V81H22ZM22 71C22 68.8757 23.76 67 26 67V65C22.615 65 20 67.8118 20 71H22ZM26 67H46V65H26V67ZM46 67C48.104 67 50 68.896 50 71H52C52 67.7915 49.2085 65 46 65V67ZM50 71V81H52V71H50ZM51 82H63.5V80H51V82ZM63.5 82C68.1012 82 72 78.2846 72 73.5H70C70 77.1529 67.0238 80 63.5 80V82ZM72 73.5V72.0938H70V73.5H72ZM72 72.0938C72 60.6258 63.3131 51.4622 52.1962 50.3178L51.9913 52.3072C62.1244 53.3503 70 61.6867 70 72.0938H72ZM43.5 70C41.5018 70 40 71.7383 40 73.5H42C42 72.7617 42.6857 72 43.5 72V70ZM40 73.5C40 75.4585 41.5415 77 43.5 77V75C42.646 75 42 74.354 42 73.5H40ZM43.5 77C45.2617 77 47 75.4982 47 73.5H45C45 74.3143 44.2383 75 43.5 75V77ZM47 73.5C47 72.5498 46.535 71.6832 45.9259 71.0741C45.3168 70.465 44.4502 70 43.5 70V72C43.7998 72 44.1832 72.16 44.5116 72.4884C44.84 72.8168 45 73.2002 45 73.5H47ZM28.5 70C26.5018 70 25 71.7383 25 73.5H27C27 72.7617 27.6857 72 28.5 72V70ZM25 73.5V81H27V73.5H25ZM26 82H31V80H26V82ZM32 81V73.5H30V81H32ZM32 73.5C32 72.5498 31.535 71.6832 30.9259 71.0741C30.3168 70.465 29.4502 70 28.5 70V72C28.7998 72 29.1832 72.16 29.5116 72.4884C29.84 72.8168 30 73.2002 30 73.5H32Z"/></symbol>
```

Comment:
This adds quite a significant amount of weight to this template. Takes it up to nearly 70KB from about 7KB. How is the performance after this change?